src/graphql/data/fetchGraphQL.ts

```diff
@@ -29,16 +25,25 @@ const NFT_QUERIES = [
   'NftBalancePaginationQuery',
 ]
 
+const fetchToken = (body: string): Promise<Response> => {
+  return fetch(TOKEN_URL, { method: 'POST', body, headers: tokenHeaders })
+}
+
+const fetchNFT = (body: string): Promise<Response> => {
+  if (!NFT_URL) {
+    throw new Error('NFT URL missing from environment')
+  }
+  return fetch(NFT_URL, { method: 'POST', body, headers: nftHeaders })
+}
+
 const fetchQuery = (params: RequestParameters, variables: Variables): Promise<GraphQLResponse> => {
-  const isNFT = NFT_QUERIES.includes(params.name)
   const body = JSON.stringify({
     query: params.text, // GraphQL text from input
     variables,
   })
-  const url = isNFT ? NFT_URL : TOKEN_URL
-  const headers = isNFT ? nftHeaders : baseHeaders
-
-  return fetch(url, { method: 'POST', body, headers })
+  const isNFT = NFT_QUERIES.includes(params.name)
```

Comment:
If we have `fetchNFT` and `fetchToken`, do we need a generic `fetchQuery` anymore? There's something about `isNFT` that I don't like... 🫣

Reply:
this bool will be removed once we've launched, are not hidden behind a feature flag, and get the data from the same endpoint

Reply:
thanks for explaining that!